test/bdd/pkg/gnap/steps.go

```diff
@@ -66,44 +70,77 @@ func (s *Steps) createGNAPClient() error {
 		Transport: &http.Transport{TLSClientConfig: s.ctx.TLSConfig()},
 	}
 
-	// create key-pair
-	pub, private, err := ed25519.GenerateKey(rand.Reader)
```

Comment:
we should support ed25519, P-256, P-384 and P-521 keys

Reply:
created issue: https://github.com/trustbloc/auth/issues/221